en/mavgen_c/example_c_udp.md

```diff
@@ -31,19 +31,20 @@ The following instructions show how to build and run the example.
    You can put/generate the library wherever you like, but the build command below assumes they are located in directory named **include** below the MAVLink root directory.
    :::
 
-1. Open a terminal and navigate to [examples/linux](https://github.com/mavlink/mavlink/tree/master/examples/linux)
-1. Compile with GCC using the following command:
+2. Open a terminal and navigate to [examples/c](https://github.com/mavlink/mavlink/tree/master/examples/c)
+
+3. Compile with GCC using the following command:
 
    ```sh
-   gcc -std=c99 -I ../../include/common -o mavlink_udp mavlink_udp.c
+   gcc -std=c99 -I ../../include/common -o mavlink_udp udp_example.c
    ```
 
    ::: info
    The MAVLink header directory must be added to the include path.
    The path here assumes you are building the code from the example directory, and that have installed the headers in **mavlink/include**.
    :::
 
-1. Run the executable from the terminal:
+4. Run the executable from the terminal:
```

Comment:
FYI, this works, but so does the previous way - markdown auto-numbers

Reply:
Thanks for the clarification! While the auto-numbering in Markdown works, I've updated the enumeration to 1. 2. 3. for consistency and clarity in the raw file.

Reply:
That's fine. I tend to prefer auto numbering, because it allows me to move list items around and insert new ones as needed. Now we use prettier this is no longer as necessary.